### PR TITLE
[TextFields] Correcting color themer

### PR DESCRIPTION
**Pull request by willlarche**
This color was not supposed to be set.

### CODE REVIEW

**Review by ajsecord — APPROVED** [review]

**Comment by willlarche** [comment]
Tested locally to recreate Travis.

File: components/TextFields/src/ColorThemer/MDCTextFieldColorThemer.m [hunk 1]
```diff
@@ -49,14 +49,6 @@ + (void)applyColorScheme:(id<MDCColorScheme>)colorScheme
   if ([textInputControllerClass respondsToSelector:@selector(setActiveColorDefault:)]) {
     [textInputControllerClass setActiveColorDefault:colorScheme.primaryColor];
   }
-
-  if ([textInputControllerClass
```
Comment (romoore) [inline components/TextFields/src/ColorThemer/MDCTextFieldColorThemer.m:53]: Does the same change need to be made for `floatingPlaceholderNormalColor`? It seems like the default would not change instances that are explicitly themed.

Comment (willlarche) [reply components/TextFields/src/ColorThemer/MDCTextFieldColorThemer.m:53]: Very good catch!

Comment (willlarche) [reply components/TextFields/src/ColorThemer/MDCTextFieldColorThemer.m:53]: Of course, this whole thing makes me wonder if I should be using the secondary color at all.........

**Pull request merged**